src/main/java/com/salesforce/dynamodbv2/mt/mappers/sharedtable/SharedTableBuilder.java

```diff
@@ -318,24 +322,28 @@ public MtAmazonDynamoDbBySharedTable build() {
         setDefaults();
         withName("SharedTableBuilder");
         validate();
+        PhysicalTableManager physicalTableManager = new PhysicalTableManager(amazonDynamoDb, pollIntervalSeconds);
```

Comment:
Abstracted out physical table create/describe logic from TableMappingFactory into the new PhysicalTableManager